gcr/ansible-vault.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Ensures ansible-vault is installed within the build environment
+#
+# NOTE: This causes an error when running in Travis
+#       DO NOT USE 'set -e' in this script, otherwise all hell breaks loose,
+#       but, yaknow, without outputing the actual issue to the Travis log...
+
+# Set TOOL_ROOT, the location of the directory this script is housed in
+readonly TOOL_ROOT=$(cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+
+# Include shell helpers
+source $TOOL_ROOT/utils.sh
+
+echo_yellow "Ensuring ansible-vault is installed..."
+
+IS_INSTALLED=$(which ansible-vault)
+
+# Check if ansible-vault tools are installed
+if [ $? -ne 0 ]; then
+ echo_yellow "ansible-vault not installed, installing now..."
```

Comment:
Looks like spacing is a bit off, all of these lines have one space instead of two, like the `echo_green` line below